src/pages/iou/propTypes/index.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import CONST from '../../../CONST';
+import participantPropTypes from '../../../components/participantPropTypes';
+
+const iouPropTypes = PropTypes.shape({
+    /** ID (iouType + reportID) of the request */
+    id: PropTypes.string,
+
+    /** Amount of the request */
+    amount: PropTypes.number,
+
+    /** Currency of the request */
+    currency: PropTypes.string,
+
+    /** Description of the request */
+    comment: PropTypes.string,
+
+    merchant: PropTypes.string,
+    created: PropTypes.string,
```

Comment:
I think this one is "Date that the request was created"

Reply:
You can probably use the same description as `transactionPropTypes.js` which is:

`/** The merchant name */`

In the context of "manual request", the merchant doesn't really make much sense. It looks like we provide a "merchant" field, but never a description or directions on how to use it (from a consumer perspective).

![image](https://github.com/Expensify/App/assets/1228807/d3f8a78e-311f-4c71-8de6-1c74a3a94c23)